$MirOS: ports/lang/tcl/8.4/patches/patch-unix_tclConfig_sh_in,v 1.1 2006/11/20 21:08:11 tg Exp $
$OpenBSD: patch-tclConfig_sh_in,v 1.1 2003/11/08 04:50:58 kevlo Exp $
--- unix/tclConfig.sh.in.orig	2008-02-06 16:38:26.000000000 +0100
+++ unix/tclConfig.sh.in	2010-01-09 18:52:18.000000000 +0100
@@ -110,10 +110,6 @@ TCL_RANLIB='@RANLIB@'
 TCL_LIB_FLAG='@TCL_LIB_FLAG@'
 
 # String to pass to linker to pick up the Tcl library from its
-# build directory.
-TCL_BUILD_LIB_SPEC='@TCL_BUILD_LIB_SPEC@'
-
-# String to pass to linker to pick up the Tcl library from its
 # installed directory.
 TCL_LIB_SPEC='@TCL_LIB_SPEC@'
 
@@ -140,14 +136,6 @@ TCL_SHARED_LIB_SUFFIX='@CFG_TCL_SHARED_L
 # VERSION.  On most UNIX systems this is ${VERSION}.a.
 TCL_UNSHARED_LIB_SUFFIX='@CFG_TCL_UNSHARED_LIB_SUFFIX@'
 
-# Location of the top-level source directory from which Tcl was built.
-# This is the directory that contains a README file as well as
-# subdirectories such as generic, unix, etc.  If Tcl was compiled in a
-# different place than the directory containing the source files, this
-# points to the location of the sources, not the location where Tcl was
-# compiled.
-TCL_SRC_DIR='@TCL_SRC_DIR@'
-
 # List of standard directories in which to look for packages during
 # "package require" commands.  Contains the "prefix" directory plus also
 # the "exec_prefix" directory, if it is different.
@@ -163,16 +151,9 @@ TCL_STUB_LIB_FILE='@TCL_STUB_LIB_FILE@'
 TCL_STUB_LIB_FLAG='@TCL_STUB_LIB_FLAG@'
 
 # String to pass to linker to pick up the Tcl stub library from its
-# build directory.
-TCL_BUILD_STUB_LIB_SPEC='@TCL_BUILD_STUB_LIB_SPEC@'
-
-# String to pass to linker to pick up the Tcl stub library from its
 # installed directory.
 TCL_STUB_LIB_SPEC='@TCL_STUB_LIB_SPEC@'
 
-# Path to the Tcl stub library in the build directory.
-TCL_BUILD_STUB_LIB_PATH='@TCL_BUILD_STUB_LIB_PATH@'
-
 # Path to the Tcl stub library in the install directory.
 TCL_STUB_LIB_PATH='@TCL_STUB_LIB_PATH@'
 
